src/main/java/com/stories/API/ApiApplication.java

```diff
@@ -0,0 +1,27 @@
+package com.stories.API;	
+
+import org.bson.Document;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+@SpringBootApplication
+public class ApiApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(ApiApplication.class, args);
+		
+		 MongoClientURI connectionString = new MongoClientURI("mongodb+srv://db_user:vloUr7AakeTEdXDj@cluster0-nomfh.mongodb.net/test?retryWrites=true&w=majority");
```

Comment:
Why are you adding the mongo connection two times?

Reply:
because I needed to consult they databases